### PR TITLE
Add pagination

### PR DESCRIPTION
**Pull request by h-michael**

### CODE REVIEW

File: src/renderer/components/Header.vue [hunk 1]
```diff
@@ -11,11 +18,20 @@
 import { Component, Vue, Prop } from 'vue-property-decorator'
 import { actions } from '../store/actions';
 
-@Component({ })
+@Component({
+  props: {
```
Comment (joe-re) [inline src/renderer/components/Header.vue]: I think, props property isn't necessary inside Component decorator in case of using `@props`.
see prop decorator implementation: https://github.com/kaorun343/vue-property-decorator/blob/master/src/vue-property-decorator.ts#L65

File: src/renderer/containers/MiniPlayerContainer.vue [hunk 2]
```diff
@@ -9,7 +9,12 @@ import { actions } from '../store/actions';
 import { remote } from 'electron';
 
 @Component({
-  components: { miniPlayer }
+  components: { "mini-player": miniPlayer },
```
Comment (joe-re) [inline src/renderer/containers/MiniPlayerContainer.vue]: Is this change needed?

File: src/renderer/components/Header.vue (hunk 1 above)
Comment (h-michael) [reply src/renderer/components/Header.vue]: I understood.

**Comment by h-michael** [comment]
@joe-re I fixed it.

File: src/renderer/components/VideoList.vue [hunk 3]
```diff
@@ -6,35 +6,65 @@
      >
        <thumbnail class="thumbnail" :item="item" :mini-player-mode="miniPlayerMode"></thumbnail>
      </div>
+     <div class="pagination-container">
+       <button class="pagination" v-if="prevPageToken" v-on:click="prevPage">prev</button>
```
Comment (joe-re) [inline src/renderer/components/VideoList.vue]: We use [Photon kit](http://photonkit.com/components/) to imitate to macOS application.
Can you add class attributes `btn btn-default` ?

**Comment by h-michael** [comment]
@joe-re I fixed style.

**Comment by joe-re** [comment]
@h-michael 
Thanks for your PR. It looks good 👍  I merge it.
